IdentityCore/src/controllers/MSIDRequestControllerFactory.m

```diff
@@ -101,6 +137,95 @@ @implementation MSIDRequestControllerFactory
     return localController;
 }
 
++ (nullable id<MSIDRequestControlling>)silentControllerWithXpcForParameters:(MSIDRequestParameters *)parameters
```

Comment:
`silentControllerWithXpcForParameters` and `SilentControllerWithoutXpcForParameters` have quite some duplicate code, is there any reason why we don't refactor them to avoid the duplicate code? :)

Reply:
As discussed offline, the new method is refactored, and we will keep one running and remove the other one if everything looks good